### PR TITLE
Updated Gumby to support a single TrustChain ledger

### PR DESCRIPTION
**Pull request by devos50**
This PR updates Gumby so the existing experiments still function as expected. This PR requires https://github.com/Tribler/tribler/pull/3572 to be merged first.

### CODE REVIEW

**Comment by devos50** [comment]
This PR is ready for review. The errors are expected since it uses the Tribler version without the TrustChain changes.

File: experiments/trustchain/trustchain_module.py [hunk 1]
```diff
@@ -71,4 +76,10 @@ def request_random_signature(self):
     def request_signature_from_peer(self, peer, up, down):
         self._logger.info("%s: Requesting signature from peer: %s" % (self.my_id, peer))
         transaction = {"up": up, "down": down}
-        self.overlay.sign_block(peer, peer.public_key.key_to_bin(), transaction)
+        self.overlay.sign_block(peer, peer.public_key.key_to_bin(), block_type='test', transaction=transaction)
+
+    def should_sign(self, block):
```
Comment (ichorid) [inline experiments/trustchain/trustchain_module.py:81]: That's a really useful method! How come we never invented it before! :wink:

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
retest this please

**Review by qstokkink — APPROVED** [review]

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
The unit tests and integration tests are fixed 👍 . We should fix the pylint checker so it does not show these import errors.

**Pull request merged**